doc/cascadia/profiles.schema.json

```diff
@@ -1478,6 +1478,11 @@
           "description": "When set to true, trailing white-spaces will be removed from text in rectangular (block) selection while copied to your clipboard. When set to false, the white-spaces will be preserved.",
           "type": "boolean"
         },
+        "trimPaste": {
+          "default": true,
+          "description": "When enabled, the Terminal will automatically trim trailing whitespace characters when pasting text to the Terminal",
```

Comment:
Redundant sentence redundancy (the terminal will... the terminal)

Reply:
```suggestion
          "description": "When enabled, the Terminal will automatically trim trailing whitespace characters when pasting text",
```